sequencing-server/src/lib/codegen/CommandEDSLPreface.ts

```diff
@@ -238,6 +247,19 @@ declare global {
   type F32 = F<32>;
   type F64 = F<64>;
 
+  function VARIABLE(name: string, type: 'FLOAT' | 'INT' | 'STRING' | 'UINT' | 'ENUM'): Variable;
```

Comment:
Should this list of valid types be a const somewhere since it's reused? At least it would be easier to modify later.

Reply:
Nice idea, I just pulled it out of the interface and make it const.